test/integration/test_webrtc_api.py

```diff
@@ -0,0 +1,233 @@
+"""
+    Bandwidth
+
+    Bandwidth's Communication APIs  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Contact: letstalk@bandwidth.com
+    Generated by: https://openapi-generator.tech
+"""
+
+
+from array import array
+import os
+import unittest
+from urllib import response
+
+import bandwidth
+from hamcrest import *
+from bandwidth.api import participants_api, sessions_api
+from bandwidth.exceptions import NotFoundException, UnauthorizedException
+from bandwidth.model.create_participant_request import CreateParticipantRequest
+from bandwidth.model.create_participant_response import CreateParticipantResponse
+from bandwidth.model.publish_permissions_enum import PublishPermissionsEnum
+from bandwidth.model.device_api_version_enum import DeviceApiVersionEnum
+from bandwidth.model.participant import Participant
+from bandwidth.model.session import Session
+from bandwidth.model.subscriptions import Subscriptions
+from bandwidth.model.participant_subscription import ParticipantSubscription
+
+
+class TestSessionsApi(unittest.TestCase):
```

Comment:
Can you add docstrings to all of your functions?

Reply:
done